application/frontend/controllers/AuthController.php

```diff
@@ -86,35 +117,40 @@ public function actionLogin()
 
     public function actionLogout()
     {
-        if (\Yii::$app->user->isGuest) {
+        $accessToken = \Yii::$app->request->get('access_token');
+        if ($accessToken !== null) {
```

Comment:
It looks like the user will only be logged out if their logout HTTP request included an access_token. Is that intentional?